scripts/build_ros2_workspace.bash

```diff
@@ -5,7 +5,6 @@ set -e
 if [[ $1 == "-h" ]] || [[ $1 == "--help" ]]; then
   echo -e "Usage: build_ros2_workspace.bash [option...] \t This script builds px4_ros_com workspace for ROS 2" >&2
   echo
-  echo -e "\t--add_ros1_bridge \t Clone ros1_bridge. To be used in a further build in case a ROS1 workspace is used. Check 'build_all' or 'build_ros1_bridge' scripts."
   echo -e "\t--ros_distro \t\t Set ROS 2 distro name (dashing|eloquent|foxy|galactic|humble|rolling). If not set, the script will set the ROS_DISTRO env variable based on the Ubuntu codename"
```

Comment:
Wondering if everything here actually still works (or there is an intent to make work) with pre-Foxy versions. Might save some future headaches to draw a line in the sand here for which distros are expected/tested to work with PX4.

Reply:
I was personally planning to draw the line at Foxy, but we can be flexible if there truly is a compelling reason.